tests/behaviour/inspectcli.feature

```diff
@@ -319,7 +319,7 @@ Feature: Behaviour testing of csvcubed inspect.
         - The data cube has the following value counts:
             - Value counts broken-down by measure and unit (of measure):
                        Measure                                          Unit  Count
-        gas emissions(gwp-ar4) millions of tonnes of carbon dioxide (mt co2)  41508
+        gas emissions(gwp-ar4) final-uk-greenhouse-gas-emissions-national-statistics-1990-to-2019.csv#unit/millions-of-tonnes-of-carbon-dioxide-mt-co2  41508
```

Comment:
This test change is highlighting that we were missing the functionality to look up the label from the unit's URI.

Reply:
Add this to the follow on ticket.